src/js/components/Data/stories/Complex.js

```diff
@@ -76,7 +85,8 @@ export const Complex = () => (
       status="info"
       message="Data is in 'beta'. The API surface is subject to change."
     />
-    <Data data={data} properties={properties} toolbar>
+    <Data data={data} properties={properties} updateOn="change">
+      <DataFilters layer />
```

Comment:
<img width="1435" alt="image" src="https://user-images.githubusercontent.com/23144209/231805862-f365e6d8-ce2f-4a3f-a29c-ac44ca933878.png">
I realized after I just tried testing the original layout -- in fullscreen interaction, there is extremely minimal scroll space for the Select options, and `<DataFilters layer />` somewhat solves the problem. I think this might be an issue unrelated to this PR?

Reply:
I'm fine adjusting this story to the DataFilters layer, but I'd probably recommend we don't add "updateOn=change" in that case.

Reply:
Thanks for clarifying! Will make that final edit.